src/index.js

```diff
@@ -0,0 +1,200 @@
+"use strict";
+
+const state = {
+    temperature: 70,
+    emoji: null,
+    sky: "😎🌈☁️🌧️😎🌈☁️🌧️😎🌈☁️🌧️😎🌈☁️🌧️😎🌈☁️",
+    skyEmoji: null,
+    lat: null,
+    lon: null
+};
+
+// DOM elements
+const temperatureElement = document.getElementById('temperature');
+const increaseBtn = document.getElementById('increase-btn');
+const decreaseBtn = document.getElementById('decrease-btn');
+const landscapeImg = document.getElementById('landscape-img');
+const emojiElement = document.getElementById('emoji');
+const cityElement = document.getElementById('city');
+const resetCityButton = document.getElementById('reset-city');
+const skyElement = document.getElementById('sky');
+const skyEmojiElement = document.getElementById('skyEmoji');
+const checkCurrentTempElement = document.getElementById('check-current-temp');
```

Comment:
Pieces of code like this are usually found in an event handler that runs on the event `DOMContentLoaded`, basically once the page is html is loaded on the webpage you will run a bunch of initial logic such as grabbing elements and adding event handlers to them. It will look something like this:

```javascript
document.addEventListener("DOMContentLoaded", function () {
    const increaseTemp = document.querySelector("#increase-temp");
    const decreaseTemp = document.querySelector("#decrease-temp");
    const displayTemp = document.querySelector("#display-temp");
    const resetButton = document.querySelector("#reset-button"); 
    const searchButton = document.querySelector("#search-button");
    const cityName = document.getElementById("city-name");
    const cityInput = document.getElementById("city-input");
    const selectSky = document.querySelector("#sky-dropdown");
    const result = document.querySelector("#sky");
    const landscape = document.querySelector("#landscape");
    
    increaseTemp.addEventListener("click", increaseTemperature);
    decreaseTemp.addEventListener("click", decreaseTemperature);
    ...
}
```

This is also usually found at the bottom of the file too, that way you can put all your function definitions for event handlers above it!